event-bus/src/lib.rs

```diff
@@ -5,6 +5,8 @@ use prometheus::{opts, register_int_counter_vec_with_registry, IntCounterVec, Re
 mod kafka;
 mod sqs;
 
+pub const KAFKA_BOOTSTRAP_SERVERS: &str = "localhost:9092";
```

Comment:
With this as a const, I'm guessing `all` is mostly a developer tool? And we would expect that it would only work with a local minio/kafka instance?

Reply:
The way I see it @jcrossley3 is that this would replace the commands you'd need for doing local development and running integration tests. After starting the compose.yml, you just need to run `cargo run -p trust all` to get a working system.